package.json

```diff
@@ -35,8 +35,8 @@
   "devDependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
     "@edx/browserslist-config": "1.2.0",
-    "@openedx/frontend-build": "14.0.3",
-    "@openedx/paragon": "22.3.1",
+    "@openedx/frontend-build": "github:openedx/frontend-build#abdullahwaheed/datadog-logging",
```

Comment:
Are you planning to merge this change to master? Or just use this branch on an MFE for testing first?

Reply:
no, we'll first merge it in alpha to test it